ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,464 +1,476 @@
 {# eos - router bgp #}
-{% if router_bgp is defined and router_bgp is not none %}
+{% if router_bgp is arista.avd.defined %}
```

Comment:
change to `if router_bgp.as is arista.avd.defined`

Reply:
agreed updated.

Reply:
Not really. Since router_bgp.as is needed for the rest of the configuration, we should replace this outmost if with the deep check for `router_bgp.as` Otherwise missing AS will now produce invalid config since `router bgp 123` is not rendered, but all following config could be.

Reply:
understood, updated!